Client/src/Utils/Theme/constants.js

```diff
@@ -108,6 +108,12 @@ const semanticColors = {
 			dark: paletteColors.green900,
 		},
 	},
+	neuter: {
```

Comment:
Is this meant to be "neutral"?

Don't often see neuter used in this context, more often in [this context](https://en.wikipedia.org/wiki/Neutering?wprov=sfla1) 😂

Reply:
OMG YES xD